_topic_maps/_topic_map.yml

```diff
@@ -3541,6 +3541,8 @@ Topics:
     File: serverless-functions-setup
   - Name: Getting started with functions
     File: serverless-functions-getting-started
+  - Name: Building and deploying on-cluster functions
```

Comment:
Here I'm just thinking a bit about "on-cluster functions", it that may mislead interpretation of users.. like "regular functions" vs "on-cluster functions" (but at the end all funtions runs on cluster)... Maybe "On-Cluster building and deploying functions" ... but I'm unsure .. wdyt?